contracts/SecurityTokenRegistry.sol

```diff
@@ -83,6 +95,14 @@ contract SecurityTokenRegistry is Ownable, ISecurityTokenRegistry, Util {
         protocolVersionST[_version] = _stVersionProxyAddress;
     }
 
+    /**
+     * @dev set the ticker registration fee in POLY tokens
+     * @param _registrationFee registration fee in POLY tokens (base 18 decimals)
+     */
+    function setPolyRegistrationFee(uint256 _registrationFee) public onlyOwner {
```

Comment:
change the function name as we are changing the registration fee, not setting the fee for one time
it should be named as `changePolyRegistrationFee` and also add the event for logging the same. with old value and new value as the arguments of the event.